cve_bin_tool/checkers/__init__.py

```diff
@@ -1,5 +1,6 @@
 """ CVE Checkers """
 __all__ = [
+    "wireshark",
```

Comment:
Please alphabetize this.  I'm not sure this file matters as code any more but that's no reason not to try to keep it clean for human readers!

Reply:
Sure, made the required changes.